spec/models/ext_management_system_spec.rb

```diff
@@ -396,6 +396,91 @@
     end
   end
 
+  context "#pause!" do
+    it 'disables an ems with child managers and moves parent to maintenance zone' do
+      zone  = FactoryGirl.create(:zone)
+      ems   = FactoryGirl.create(:ext_management_system, :zone => zone)
+      child = FactoryGirl.create(:ext_management_system, :zone => zone)
+      ems.child_managers << child
+
+      ems.pause!
+
+      expect(ems.enabled).to be_falsy
+      expect(ems.zone).to eq(Zone.maintenance_zone)
+      expect(ems.zone_before_pause).to eq(zone)
+
+      child.reload
+      expect(child.enabled).to be_falsy
+    end
+  end
+
+  context "#resume" do
+    it "enables an ems with child managers and move parent from maintenance zone" do
+      zone = FactoryGirl.create(:zone)
+      ems = FactoryGirl.create(:ext_management_system,
+                               :zone_before_pause => zone,
+                               :zone              => Zone.maintenance_zone,
```

Comment:
When this method no longer automatically creates the maintenance zone as a side effect, you'll need to create it during the test

Reply:
response above in creating

Reply:
:+1: https://github.com/ManageIQ/manageiq/pull/17452/files#diff-1f5be100b3b69417b854ff9a45a24fa3R420